extensions/gluonTS/src/main/java/ai/djl/gluonTS/SampleForeCast.java

```diff
@@ -0,0 +1,109 @@
+/*
+ * Copyright 2022 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"). You may not use this file except in compliance
+ * with the License. A copy of the License is located at
+ *
+ * http://aws.amazon.com/apache2.0/
+ *
+ * or in the "license" file accompanying this file. This file is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES
+ * OR CONDITIONS OF ANY KIND, either express or implied. See the License for the specific language governing permissions
+ * and limitations under the License.
+ */
+package ai.djl.gluonTS;
+
+import ai.djl.ndarray.NDArray;
+import ai.djl.ndarray.NDManager;
+
+import java.time.LocalDateTime;
+
+/**
+ * A {@code Forecast} object, where the predicted distribution is represented internally as samples.
+ */
+public class SampleForeCast extends ForeCast {
+
+    protected static NDManager samplesManger = NDManager.newBaseManager();
```

Comment:
We should not have static NDManager, that will cause memory leak

Reply:
Thanks for this hint! But, actually I don't know how to manage the lifetime of the NDArray in the Forecast, or can I make the Forecast class implement the NDResource interface?

Reply:
Well, I gues may be to let the SampleForeCast to implement AutoCloseable and new a NDManager for every SampleForecast instance is a feasible way?